### PR TITLE
add .post0 installation for aws

### PR DESCRIPTION
**Pull request by StevenJokes**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1356/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1356/

**Comment by mli** [comment]
Job d2l-en/PR-1356/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1356/

File: chapter_appendix-tools-for-deep-learning/aws.md [hunk 1]
```diff
@@ -217,14 +217,14 @@ After creating the `d2l` environment, activate it and install `pip`.
 
 ```bash
 conda activate d2l
-conda install python=3.7 pip -y
+conda install python=3.7 pip notebook -y
```
Comment (astonzhang) [inline chapter_appendix-tools-for-deep-learning/aws.md:220]: no need for users, jupter is in https://github.com/d2l-ai/d2l-book/blob/master/setup.py#L5

**Comment by mli** [comment]
Job d2l-en/PR-1356/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1356/

**Comment by mli** [comment]
Job d2l-en/PR-1356/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1356/

**Comment by mli** [comment]
Job d2l-en/PR-1356/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1356/

**Comment by astonzhang** [comment]
This file has already shown post0

**Pull request closed without merging**